src/vs/code/electron-main/window.ts

```diff
@@ -536,6 +561,21 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 			this.currentMenuBarVisibility = newMenuBarVisibility;
 			this.setMenuBarVisibility(newMenuBarVisibility);
 		}
+
+		const config = this.configurationService.getValue<IWorkbenchEditorConfiguration>();
+		this._win.removeAllListeners('swipe');
+		this._win.removeAllListeners('app-command');
+		if (config && config.workbench && config.workbench.editor) {
+			let swipeConfig = config.workbench.editor.swipeToNavigate;
+			if (swipeConfig && swipeConfig !== 'off') {
```

Comment:
I am not 100% sure, but it seems the integration tests run with certain values within the config as null.
I know that `workBench` is null, so calling `editor` on it failed here https://dev.azure.com/vscode/a4cdce18-a05c-4bb8-9476-5d07e63bfd76/_build/results?buildId=29436

So with that I assume the type definitions in TS are invalid, since they don't indicate that they're optionals. 
I'm checking every single value here just to see what the CI is saying.